dayPlanner.js

```diff
@@ -14,84 +14,39 @@ for (var i = 1; i <= 15; i++) {
             if (element.style.display === "none" || element.style.display === "") {
                 element.style.display = "block";
             } else {element.style.display = "none";}
+            if (activeFuncNum !== null && activeFuncNum !== undefined) {
+                let deleteButton = "SİL"
+                let addButon = "EKLE";
+                let td = "td" + activeFuncNum;
+                var tdText = document.getElementById(td)
+                todoText.value = tdText.textContent;
+                addTodo.textContent = tdText.textContent ? deleteButton : addButon;

```

Comment:
- ekle butonuna bir değer atayacağız.
- `tdText`'in içeriğine bakıyoruz.
- eğer `true` ise yani bir metin varsa, `deleteButton`'u atıyoruz.
- eğer `false` ise yani bir metin yoksa, `addButton`'u atıyoruz.